### ISSUE TITLE
Whiteboard is not working on OBS version 28

### ISSUE THREAD

**Issue by peow** [katarai/obs-whiteboard-lua#2]
Whiteboard is not working on OBS version 28, could you fix it please?

**Comment by XacerPT** [comment]
yup, dont work for me too in obs version 29, it would be great if it were updated

**Comment by katarai** [comment]
It seems like some folks got it working on OBS 28 (not sure about 29). Not sure what the deciding factor is.

I'll look into it when I have some time, but that may be in a month or two, as I'm very busy at the moment. :(

**Comment by katarai** [comment]
Could not reproduce. Other users have reported similar issues in v29 (on the OBS forums), but I have been unable to reproduce the issue even with clean installs on different PCs.

If there's any additional information about what may be causing the issue, please let me know.

**Comment by HeyItsLollie** [comment]
Recently moved to a new PC, so I had to reinstall this plugin. Whiteboard initially didn't appear to be working on 29.1.3, but I was able to get it working after:
- Setting my hotkeys
- Using my hotkeys to toggle the Color and the Eraser until everything started working again

I believe the issue was that by default, Whiteboard was using either the eraser, or some sort of "null" color brush that wasn't producing any color.

**Comment by katarai** [comment]
> 

Interesting, thanks for the info. I'll do some more digging this weekend.

**Comment by katarai** [comment]
Odd, I don't see anything in the code that suggests the initial colors or tools would be set incorrectly. I wonder if it's somehow initializing incorrectly or if some setting is getting saved between installs.

**Comment by Elementropy** [comment]
Still not working on OBS v30.1.2

I followed all instructions, set hot keys and used them to toggle all options multiple times. Drawing on the projector simply doesn't work anymore. Any news on a patch?

Thanks and have a good day.
